stackit/internal/services/argus/instance/resource.go

```diff
@@ -64,6 +66,100 @@ type Model struct {
 	OtlpTracesURL                      types.String `tfsdk:"otlp_traces_url"`
 	ZipkinSpansURL                     types.String `tfsdk:"zipkin_spans_url"`
 	ACL                                types.Set    `tfsdk:"acl"`
+	AlertConfig                        types.Object `tfsdk:"alert_config"`
+}
+
+var mockAlertConfig = alertConfigModel{
+	Receivers: types.ListValueMust(types.ObjectType{AttrTypes: receiversTypes}, []attr.Value{
+		types.ObjectValueMust(receiversTypes, map[string]attr.Value{
+			"name":             types.StringValue("change-me"),
+			"email_configs":    types.ListValueMust(types.ObjectType{AttrTypes: emailConfigsTypes}, []attr.Value{}),
+			"opsgenie_configs": types.ListValueMust(types.ObjectType{AttrTypes: opsgenieConfigsTypes}, []attr.Value{}),
+			"webhooks_configs": types.ListValueMust(types.ObjectType{AttrTypes: webHooksConfigsTypes}, []attr.Value{}),
+		}),
+	}),
+	Route: types.ObjectValueMust(routeTypes, map[string]attr.Value{
+		"receiver": types.StringValue("change-me"),
+	}),
+}
```

Comment:
It looks a little more natural to define the mock in the actual type from the SDK, instead of in terms of the model. Or does this makes it easier in any way?

Reply:
Also because if you remove the `...ValueMust` the function will possibly return an error and will make it a bit weirder

Reply:
The mock configuration is only used for Terraform models, which means that it is not very practical to set it to an SDK payload, since I would always have to convert it. 

Also, setting it to the locally defined `alertConfigModel` allows it to be directly called in the `Create` method and the conversion is less complex in the `alertConfigModel` -> `basetypes.ObjectValue` direction than in reverse. But for this I can see a case for either type, let me know if you have a strong opinion